From 7ec819745b5d9ad21a6aaf84a311b87fe7a41e23 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Oliver=20St=C3=A4bler?= <oliver.staebler@bytesatwork.ch>
Date: Mon, 8 Mar 2021 15:28:20 +0100
Subject: [PATCH] arm: dts: stm32mp157c-bytedevkit: Move memory node to module

It makes more sense to have the node 'memory' in the dtsi for the SOM.
---
 arch/arm/dts/stm32mp157c-bytedevkit.dts     | 4 ----
 arch/arm/dts/stm32mp157c-byteengine-m5.dtsi | 4 ++++
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/dts/stm32mp157c-bytedevkit.dts b/arch/arm/dts/stm32mp157c-bytedevkit.dts
index fcf7875231..7e47a9e689 100644
--- a/arch/arm/dts/stm32mp157c-bytedevkit.dts
+++ b/arch/arm/dts/stm32mp157c-bytedevkit.dts
@@ -20,10 +20,6 @@
 		stdout-path = "serial0:115200n8";
 	};
 
-	memory@c0000000 {
-		reg = <0xc0000000 0x20000000>;
-	};
-
 	usb_phy_tuning: usb-phy-tuning {
 		st,hs-dc-level = <2>;
 		st,fs-rftime-tuning;
diff --git a/arch/arm/dts/stm32mp157c-byteengine-m5.dtsi b/arch/arm/dts/stm32mp157c-byteengine-m5.dtsi
index 5817c8cc40..30259b58d0 100644
--- a/arch/arm/dts/stm32mp157c-byteengine-m5.dtsi
+++ b/arch/arm/dts/stm32mp157c-byteengine-m5.dtsi
@@ -14,6 +14,10 @@
 / {
 	compatible = "batw,m5", "st,stm32mp157";
 
+	memory@c0000000 {
+		reg = <0xc0000000 0x20000000>;
+	};
+
 	reserved-memory {
 		#address-cells = <1>;
 		#size-cells = <1>;
-- 
2.20.1

